rundeck-data-model/src/main/java/org/rundeck/app/data/model/v1/pluginMeta/RdPluginMeta.java

```diff
@@ -0,0 +1,12 @@
+package org.rundeck.app.data.model.v1.pluginMeta;
+
+import java.util.Date;
+
+public interface RdPluginMeta {
+    Long getId();
+    String getKey();
+    String getProject();
+    String getJsonData();
```

Comment:
Let's define this as `Map<String,Object> getPluginData()` which is found on the PluginMeta domain class. That will allow the data to be transferred as json rather then json embedded inside json. 

It's unfortunate that we have to use a map here to hold the data, but since we truly don't know what it is and it could be anything we have to break our informal 'don't use maps in the api' rule.